docs_sphinx/user/input.rst

```diff
@@ -155,10 +155,21 @@ Poisson inputs
 If the given value for ``rates`` is a constant, then using
 ``PoissonGroup(N, rates)`` is equivalent to
 ``NeuronGroup(N, 'rates : Hz', threshold='rand()<rates*dt')`` and setting the
-group's ``rates`` attribute. If ``rates`` is a string, then it is equivalent
+group's ``rates`` attribute.
+
+If ``rates`` is a string, then it is equivalent
 to ``NeuronGroup(N, 'rates = ... Hz', threshold='rand()<rates*dt')`` with the
```

Comment:
Same as above (i.e. please put it on its own line), also I think including the colon before the unit would improve readibility a bit (i.e. `'rates = ... : Hz'`).